extensions/jackson/runtime/src/main/doc/usage.adoc

```diff
@@ -0,0 +1,73 @@
+=== Configuring the Jackson `ObjectMapper`
+
+There are a few ways of configuring the `ObjectMapper` that the `JacksonDataFormat` uses. These are outlined below.
+
+==== `ObjectMapper` created internally by `JacksonDataFormat`
+
+By default, `JacksonDataFormat` will create its own `ObjectMapper` and use the various configuration options on the `DataFormat`
+to configure additional Jackson modules, pretty printing and other features.
+
+==== Custom `ObjectMapper` for `JacksonDataFormat`
+
+You can pass a custom `ObjectMapper` instance to `JacksonDataFormat` as follows.
+
+[source,java]
+----
+ObjectMapper mapper = new ObjectMapper();
+JacksonDataFormat dataFormat = new JacksonDataFormat();
+dataFormat.setObjectMapper(mapper);
+----
+
+==== Using the Quarkus Jackson `ObjectMapper` with `JacksonDataFormat`
+
+The Quarkus Jackson extension exposes an `ObjectMapper` CDI bean which can be discovered by the `JacksonDataFormat`.
```

Comment:
It could be helpful to mention that you need to include `io.quarkus:quarkus-jackson` in the dependencies.

Reply:
Not required because it comes for free from `camel-quarkus-jackson`.

https://github.com/apache/camel-quarkus/blob/main/extensions/jackson/runtime/pom.xml#L46-L49